regression/pages/studio/pages_page_studio.py

```diff
@@ -143,3 +144,67 @@ def click_and_verify_see_an_example(self):
         self.wait_for_element_visibility(
             'img[alt="Preview of Pages in your course"]', 'Pop up visibility'
         )
+
+    def click_hide_show_toggle(self, index=0):
+        """
+        Clicks hide/show toggle button
+        """
+        toggle_checkbox_css = '.toggle-checkbox'
+        toggle_checkbox = self.q(css=toggle_checkbox_css).results[index]
+        ActionChains(self.browser).move_to_element(
+            toggle_checkbox
+        ).click().perform()
+        wait_for_notification(self)
+        return self.browser.execute_script(
```

Comment:
Well it was a last minute whim. The query was complex, and I thought it would be better to do it directly in jQuery.